workshops/build-a-binderhub/binderhub_resources/hackmd-template.md

```diff
@@ -0,0 +1,73 @@
+# Notes from Build a BinderHub workshop
```

Comment:
Would it be useful if the header is "Template for collaborative notes in a Build a BinderHub workshop"? It was unclear when I started to read this document that this was not your or instructor personal notes but a collaborative notes for the workshop attendees.